front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -24,7 +26,10 @@ import SimulationReportSheet from './SimulationReportSheet';
 import StcdmResultsTable from './StdcmResultsTable';
 import StdcmSimulationNavigator from './StdcmSimulationNavigator';
 
-type StcdmResultsProps = {
+const SPEED_SPACE_CHART_HEIGHT = 521.5;
+const HANDLE_TAB_RESIZE_HEIGHT = 20;
+
+type StcdmResultsV2Props = {
```

Comment:
Stdcm v1 has recently been removed so we don't use v2 anymore, you need to leave it like it was

```suggestion
type StcdmResultsProps = {
```